src/lib/api/api.go

```diff
@@ -174,8 +174,10 @@ func (a *API) Query(w http.ResponseWriter, r *http.Request) {
 		startTimestamp, endTimestamp int64
 		err                          error
 	)
-	timeSeriesPath := r.URL.Query().Get("timeSeriesPath")
-
+	hash := r.URL.Query().Get("hashKey")
+	Type := r.URL.Query().Get("type")
+	var timeSeriesPath string
+	var newBlocks []tsdb.Block
```

Comment:
These two are used down. I suggest declaring them close to their usecase.

Reply:
I will do these changes.